src/store/modules/attestations.js

```diff
@@ -40,6 +40,7 @@ const actions = {
               }
             });
           });
+          attestationsArr.reverse();
```

Comment:
I would prefer if the order of the attestations on the app state stay same as what the node responsed, the order could be changed just here actually: https://github.com/obytescript/obyte.io/blob/c8a3624a447f26726c7af9a4e7a1db363aedcf80/src/views/ProfileAttestations.vue#L25 with a deep clone that can be done like this `JSON.parse(JSON.stringify(attestations))`